community/openebs/templates/service-openebs-prometheus.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.policies.monitoring.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "openebs.fullname" . }}-prometheus-service
+  labels:
+    app: {{ template "openebs.name" . }}
+    chart: {{ template "openebs.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: NodePort
+  ports:
+  - port: {{ .Values.prometheus.ports.externalPort }} # this Service's port (cluster-internal IP clusterIP)
+    targetPort: {{ .Values.prometheus.ports.internalPort }} # pods expose this port
+    nodePort: {{ .Values.prometheus.ports.nodePort }}
+    # Note that this Service will be visible as both NodeIP:nodePort and clusterIp:Port
+  selector:
+    app: {{ template "openebs.name" . }}
+    release: {{ .Release.Name }}
+    component: prometheus
+{{- end }}

```

Comment:
Removed the built-in prometheus and grafana instances from OpenEBS chart

Reply:
Thanks @muratkars -- with the built-in prometheus and grafana removed from the chart, the one thing you'll want to do now is add a kubernetes service that exposes the port where your exporter container is publishing metrics, and use the annotation mentioned in the comment above, so that your metrics will be picked up by the platform's prometheus.